src/test/java/com/wikia/webdriver/testcases/mercurytests/RecentWikiActivityTest.java

```diff
@@ -18,6 +18,7 @@
 public class RecentWikiActivityTest extends NewTestTemplate {
 
   @Test(groups = "mercury_recentWikiActivity_anonNavigateToRWAfromMenu")
+  @Execute(mockAds = "true")
```

Comment:
@Wikia/adeng @SebastianMarzjan tests for interstitial are now expanded on verifying does close button work.